### ISSUE TITLE
MySQL Database Errors

### ISSUE THREAD

**Issue by 3ssej** [DevotedMC/NameLayer#30]
Hello.
You may remember me - I sort of wasted your time because I got my builds from the wrong place - but I can assure you I've gotten them from the right place now.
Anyways, I'm trying to use NameLayer again but I am having trouble getting it to load the first time.
Here is my log file:
https://pastebin.com/KzStAbcL
What can I do to solve this?
Thanks again for your help and dedication!

**Comment by ProgrammerDan** [comment]
Check with your DBMS hosting provider -- it seems the user you configured doesn't have permission to modify the database structure?

**Comment by 3ssej** [comment]
Opened a ticket. They had me try a couple of things but now they are looking into it further. Thanks for your help thus far! I'll let you know if it doesn't work after they try fixing it.

**Comment by ProgrammerDan** [comment]
Sounds good. If they aren't able to get you running, I can probably provide you with a schema layout they could install for you. Let me know.

**Comment by 3ssej** [comment]
They said that the user has full permission to the database, so I'm not sure what else is wrong. Let me try it real quick with another plugin and I'll see if I get the same error.

**Comment by ProgrammerDan** [comment]
it might just be the ALTER command itself (perhaps a v. incompatibility)  --  what version of MySQL are you using?

**Comment by 3ssej** [comment]
Yeah, CoreProtect _seems_ to work okay with the database. I am asking what version they are using.

**Comment by 3ssej** [comment]
They just keep saying that it isn't relevant. I'm going to try to use a database hosted elsewhere accessible externally and see if that works.

**Comment by 3ssej** [comment]
Alright, well I tried using another database, and while I did get further in the migration, it failed at Migration 7.
Console: https://pastebin.com/D622HVuw
Looks like another permissions issue though.
Thanks again for all of your help.

**Comment by 3ssej** [comment]
I tried a third database and it migrated successfully. Please feel free to close this :D
